### PR TITLE
Cypress: integrate additional volume cases

### PR DESCRIPTION
**Pull request by DaiYuzeng**
- https://harvester.github.io/tests/manual/volumes/support-volume-hot-unplug/ 
- https://harvester.github.io/tests/manual/volumes/edit-volume-increase-size-form/ 
- https://harvester.github.io/tests/manual/volumes/detach-volume-from-virtual-machine/

### CODE REVIEW

File: cypress/utils/components/cru-resource.po.ts [hunk 1]
```diff
@@ -206,9 +212,52 @@ export default class CruResourcePo extends PagePo {
     this.save()
   }
 
+  // make sure that current page is the list page.
+  public search(queryString: any) {
+    cy.get('.sortable-table-header .search-box').should('be.visible');
+    cy.get(this.searchInput).type(queryString);
```
Comment (WuJun2016) [inline cypress/utils/components/cru-resource.po.ts:218]: cy.get(this.searchInput).clear().type(queryString);

**Review by WuJun2016 — COMMENTED** [review]
code LGTM， 
Please test first to ensure that the test case can pass. thanks

File: cypress/utils/components/cru-resource.po.ts (hunk 1 above)
Comment (DaiYuzeng) [reply cypress/utils/components/cru-resource.po.ts]: Done.

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Pull request merged**